src/test/java/dwbh/api/services/VotingServiceTests.java

```diff
@@ -190,7 +191,7 @@ void testCreateVoteFailsBecauseUserHasAlreadyVoted() {
     // and: just one checker has been called an no vote has been created
     verify(votingRepository, times(1)).findVoteByUserAndVoting(any(), any());
     verify(votingRepository, times(0)).hasExpired(any());
-    verify(votingRepository, times(0)).findGroupByUserAndVoting(any(), any());
+    // verify(votingRepository, times(0)).findGroupByUserAndVoting(any(), any());
```

Comment:
Why not removing it up front ?

Reply:
Because if we set the cache, we will be able to check it again, and I don't want to lose it... But yes, commented code is BAAAAD